projects/control-service/projects/pipelines_control_service/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
```

Comment:
Why are the gradle jar and properties in source controL? I think those are dynamically downloaded on demand when needed.